### PR TITLE
2.0.26-lto

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

**Comment by dschuff** [comment]
This change sets the LTO version as "latest" so it will effectively be the default this time. Not sure if we want that or not.
Also I have no idea why the script re-sorted the other LTO builds when I ran it?

**Comment by dschuff** [comment]
Question about the bazel failure. I assume we don't actually want to update the bazel file in this case? It doesn't have the other LTO revisions.

File: emscripten-releases-tags.txt [hunk 1]
```diff
@@ -9,17 +9,18 @@
...
     "2.0.23": "77b065ace39e6ab21446e13f92897f956c80476a",
+    "2.0.23-lto": "3f6dbb899f61fab52e4574beb4f7c8382658aa20",
     "2.0.22": "6465a9acb820207acf7da44661a7de52d0a1ae3c",
     "2.0.21": "72f4ec97fbc7ec16c15ae68a75b0a257b2835160",
-    "2.0.20-lto": "d1b26cd17e51c5c705eea69b9545975e3705c058",
     "2.0.20": "e0c15cd14170f407a9eb27fcbad22931dc67feb7",
-    "2.0.19-lto": "4487f6c5107e7882ae2bad6d26c34ffdceb713f0",
+    "2.0.20-lto": "d1b26cd17e51c5c705eea69b9545975e3705c058",
     "2.0.19": "9b9ff2dabfb4a7fbacbc004c0bead12a60f9d05c",
+    "2.0.19-lto": "4487f6c5107e7882ae2bad6d26c34ffdceb713f0",
```
Comment (sbc100) [inline emscripten-releases-tags.txt:23]: I normally revert these re-ordering.. we should investigate why they happen.

**Review by sbc100 — APPROVED** [review]

**Review by kripken — APPROVED** [review]

File: emscripten-releases-tags.txt (hunk 1 above)
Comment (dschuff) [reply emscripten-releases-tags.txt:23]: Should we just do the reordering in a different PR? does it actually matter?

Comment (sbc100) [reply emscripten-releases-tags.txt]: I just depends if the re-ordering is stable, or if it going to change ever time we run the script on different machine.  I never got around to figuring out if its just dict-ordering or something else.    I think recent python versions have stable dict order but I don't know if that is all this is.

**Pull request merged**